doc/TESTING.md

```diff
@@ -9,24 +9,24 @@ To run tests:
 ```
 
 
-To watch the tests:
+
+Also test watcher can be launched. It will re-run the entire test suite when tests changed.
```

Comment:
"when tests changed.", this should be "when any file is modified", no? or at least relevant files